meson.build

```diff
@@ -409,21 +417,35 @@ executable('ratbagd',
 install_man('ratbagd/ratbagd.8')
 
 #### ratbagd_devel ####
+#
+# A development ratbagd server that owns a different name on the bus (named
+# after the git sha). Do not use directly, this server is used by
```

Comment:
Not anymore :laughing:.

Reply:
fixed! thanks for the review. I also squashed in the removal of the git-sha-generating script.